hw1/01-fizzbuzz.js

```diff
@@ -8,9 +8,27 @@ Write a program that writes all the numbers from 1 to 100, with some exceptions:
 Use console.log() to write the proper output to the command line.
 
 **/
+function processFizzbuzz(num) {
+  let output = ''
+  if ((num % 3) === 0) {
```

Comment:
logic is correct, but it is complex to uderstand at first glance and I think using if, else if and else statement is better than using multiple if statements